locale/en/hammer-cli-foreman-webhooks.po

```diff
@@ -1,23 +1,24 @@
-# English translations for hammer-cli-foreman-webhooks package.
-# Copyright (C) 2023 THE PACKAGE'S COPYRIGHT HOLDER
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
 # This file is distributed under the same license as the hammer-cli-foreman-webhooks package.
-# FIRST AUTHOR <EMAIL@ADDRESS>, 2023.
+# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: hammer-cli-foreman-webhooks 0.0.4\n"
 "Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2023-07-12 17:00+0000\n"
 "PO-Revision-Date: 2023-07-12 17:00+0000\n"
 "Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
-"Language-Team: English\n"
-"Language: en\n"
+"Language-Team: LANGUAGE <LL@li.org>\n"
+"Language: \n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
-"Plural-Forms: nplurals=; plural=;\n"
-"\n"
+"Plural-Forms: nplurals=INTEGER; plural=EXPRESSION;\n"
 
-msgid "File containing X509 Certification Authorities concatenated in PEM format"
+msgid ""
+"File containing X509 Certification Authorities concatenated in PEM format"
```

Comment:
Uhmm, this looks suspicious

Reply:
I've noticed that too, but I've seen that few times, so I mostly ignore that. Maybe I shouldn't...

On the other hand, we pulled fully translated file for Georgian language, where it looks similar.

Reply:
This seems to be a multiline syntax so all should be good here